SETUP.md

```diff
@@ -105,7 +114,7 @@ These steps are for OSX devices, including Apple Macbooks running on [Apple Sili
 1. Install **rbenv** via `brew install rbenv`
 
 1. Install **Ruby**
-    1. For non-M1 systems, running `rbenv install` from the project root directory should be sufficient
+    1. For non-M1 systems (including M1 systems using Rosetta), running `rbenv install` from the project root directory should be sufficient.
```

Comment:
I didn't know that M1 systems using Rosetta didn't need the following special config. Thanks for clarifying this.